docs/en/stack/security/authentication/token-authentication-services.asciidoc

```diff
@@ -0,0 +1,57 @@
+[role="xpack"]
+[[token-authentication-services]]
+=== Token-based authentication services
+
+The {stack-security-features} authenticate users by using one or more token-based
+authentication services and <<realms,realms>> defined in a _realm chain_. The
```

Comment:
Does this mean that you specify the token-based services in the realm chains too? If so, I think this page should be updated with an example:
https://www.elastic.co/guide/en/elastic-stack-overview/master/realm-chains.html